### DISCUSSION TITLE
Configurable disabling of retrospective reflection anonymity

### DISCUSSION THREAD

**Discussion by jordanh** [ParabolInc/parabol#5499] [New features]
See also Issue: https://github.com/ParabolInc/parabol/issues/2806

~27 comments as of this posting

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/61469401) writes: 

> We just had our first regular retro using parabol but were struggling hard with the owner of a card. With our previous tool goretro I could just click on the icon of a person who then could pitch their topics. I couldn't find a way to know who wrote what in the "Group" section. We ended up putting name prefixes to some cards which wasted a lot of time and didn't finish. And once voted we completely lost the overview. So, how do we get a "creator label" to every card and the column it was in previously?

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/64282151) 🔒 writes:

> Retro cards are anonymous, but we have each person explain their card and give more context. It would be nice to have the name of the person who created the card visible somewhere so I don't have to ask each time who wrote the card.

**Comment by drewhousman1010** [comment]
A[ user](https://app.hubspot.com/contacts/3888472/contact/66664751) 🔒 writes: 

> Is there a way or any plans of having the name shown in the card that they wrote? Coz right now we have to add -<name> to the end of each card just to show right away who wrote it

**Comment by avivapinchas** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/72116301) 🔒 writes: 
> I'm not a fan of anonymous cards for the retro. It doesn't reflect an environment of high trust.

**Comment by charlesmathias** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/69825751) 🔒 writes:
> It would be nice to have the possibility of having the cards not anonymous in the “Stop/Start/Continuous” type of meeting. I understand that it is designed for anonymous feedback but this feature would fir better our process, where each card has an owner and we have a step to explain it to the others in the retro.

**Comment by treyparabol** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/69596351) 🔒 writes:

> I have just started using Parabol and I like it (almost). I just wanted to know if there was a way to view the author of the cards during the retrospectives so that they can read out their cards during the Group phase. Right now everyone has to scan all the posts to find their own.
